src/components/common/Graph.js

```diff
@@ -1,12 +1,12 @@
 import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-
+//Alexander changes
```

Comment:
Not sure that we need these comments? I don't think its a huge deal but github can make this kind of attribution for us

Reply:
^agreed, we could delete these through github directly without another commit push